specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/managedDatabases.json

```diff
@@ -0,0 +1,717 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
```

Comment:
>paths [](start = 3, length = 5)

I noticed that you removed GET /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/databases/{databaseName}/restoreDetails/{restoreDetailsName} when comparing with 2018-06-01-preview version. Is that intentional?

Reply:
I did not do any manual modification on this. I ran generate swagger so that the manifest gets generated. Should I manually add it back?

Reply:
I wouldn't know. Will those be routes that you support in this API version? If yes, then you probably should investigate why your generation logic excluded it. 

---
In reply to: [343892400](https://github.com/Azure/azure-rest-api-specs/pull/7551#discussion_r343892400) [](ancestors = 343892400)